demo/demo-springmvc/springmvc-server/src/main/java/org/apache/servicecomb/demo/springmvc/server/CodeFirstSpringmvc.java

```diff
@@ -389,6 +390,23 @@ public Object testObject(@RequestBody Object input) {
 
   @PostMapping(path = "/generic")
   public Holder<User> testGeneric(@RequestBody Holder<User> input) {
+    Assert.isInstanceOf(Holder.class, input);
+    Assert.isInstanceOf(User.class, input.value);
+    return input;
+  }
+
+  @PostMapping(path = "/generic1")
+  public Generic<User> testGeneric1(@RequestBody Generic<User> input) {
+    Assert.isInstanceOf(Generic.class, input);
+    Assert.isInstanceOf(User.class, input.value);
+    return input;
+  }
+
+  @PostMapping(path = "/generic2")
+  public Generic<Generic<User>> testGeneric2(@RequestBody Generic<Generic<User>> input) {
```

Comment:
We can add more test cases such as Long, Status(enum),Date types. Because before found in highway, Long is not supported.

Reply:
done
and move generic test case to TestGeneric.java